package capkproto;

import "capk_globals.proto";

message execution_report {
	optional bytes		cl_order_id = 1;	
	optional bytes		orig_cl_order_id = 2;
	optional string		exec_id = 3;
	optional sfixed32	exec_trans_type = 4;
	optional sfixed32	order_status = 5;
	optional sfixed32	exec_type = 6;
	optional string		symbol = 7;
	optional string		security_type = 8;
	optional side_t		side = 9;
	optional double		order_qty = 10;
	optional sfixed32	ord_type = 11;
	optional double		price = 12;
	optional double		last_shares = 13;
	optional double		last_price = 14;
	optional double		leaves_qty = 15;
	optional double		cum_qty = 16;
	optional double		avg_price = 17;
	optional sfixed32	time_in_force = 18;
	optional string		transact_time = 19;
	optional string		exec_inst = 20;	
	optional sfixed32	handl_inst = 21;
	optional sfixed32	order_reject_reason = 22;
	optional double		min_qty = 23;
    optional sfixed32   venue_id = 24;
    optional string     account = 25;
    optional string     exec_ref_id = 26;
}
